projects/swift-protobuf/fuzz.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import SwiftProtobuf
+
+@_cdecl("LLVMFuzzerTestOneInput")
+public func test(_ start: UnsafeRawPointer, _ count: Int) -> CInt {
+  let bytes = UnsafeRawBufferPointer(start: start, count: count)
+  var options = BinaryDecodingOptions()
+  options.messageDepthLimit = 256
+  do {
+    let _ = try ProtobufUnittest_NestedTestAllTypes.self.init(serializedData: Data(bytes),
```

Comment:
`ProtobufUnittest_NestedTestAllTypes.self.init(` can just be `ProtobufUnittest_NestedTestAllTypes(`

Reply:
Ok thanks

Reply:
You shouldn't need the `.init` in there still.

Reply:
ok